packages/terra-search-field/tests/wdio/search-field-spec.js

```diff
@@ -21,7 +21,7 @@ Terra.describeViewports('Search Field', ['medium'], () => {
     });
 
     it('should display Search Field with scrolled text', () => {
-      expect($('input[type="search"]').isFocused()).toBeTruthy();
+      expect($('input').isFocused()).toBeTruthy();
```

Comment:
Since `type=search` has been added back these wdio spec changes can be reverted

Reply:
Added.